features/SecurityController.feature

```diff
@@ -413,6 +413,27 @@ Feature: Security Controller
       | "test-user"  |
       | "test-user2" |
 
+  @security
+  Scenario: Upsert user
+    When I successfully execute the action "security":"upsertUser" with args:
+      | _id        | "user-test"           |
+      | body       | { "content": { "profileIds": ["default"]}, "default": { "name": "default-name", "credentials": { "local": { "username": "user-test", "password": "user-test" } } } } |
```

Comment:
Instead of a huge one lined body you could build your object property by property to have more readable step case
```
| body.content.profileIds | ["default"] |
```
etc

Reply:
Not supported by cucumber, sadly :(

Reply:
Finally needs a rewrite of some test